server/middlewares/controllers/authorController.js

```diff
@@ -0,0 +1,12 @@
+const Author = require("../../models/Author");
+
+const addAuthor = async (req, res) => {
+  const { name } = req.body;
+  const response = await Author.query()
```

Comment:
we probably wanna name this variable `user`. The database returns the newly created user, not a response.